extensions/nginx/index.js

```diff
@@ -222,7 +236,10 @@ class NginxExtension extends cli.Extension {
                     this.ui.sudo(`rm -f /etc/nginx/sites-available/${confFile}`),
                     this.ui.sudo(`rm -f /etc/nginx/sites-enabled/${confFile}`)
                 ]).catch(
-                    () => Promise.reject(new cli.errors.SystemError('Nginx config file link could not be removed, you will need to do this manually.'))
+                    (error) => Promise.reject(new cli.errors.CliError({
+                        message: `Nginx config file link could not be removed, you will need to do this manually for ${confFile}.`,
```

Comment:
this should maybe be a process error instead? Also, `confFile` will be the simple configuration filename, might be worth directing the user to remove the file in `/etc/sites-available` or `/etc/sites-enabled`.

Reply:
Yeah, I know... I also think it should be a `ProcessError`, but our usage of `ProcessError` doesn't include custom help text. That's why I picked the default `CliError`. 

Certainly makes sense to lead the user to the location of `confFile` for sure 👍

Reply:
Maybe it makes sense to submit another PR that improves the `ProcessError` to deal with custom properties?

Reply:
I added a task to the existing improve error handling issue #587